### ISSUE TITLE
NumberInput - rename tokens

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/B01LER-Kitchen#534]
#### Description / User Story

After changing the structure of the component tokens within #239 , we now need to adjust the token names for the NumberInput.

----
#### Acceptance Criteria
- [ ] Design: All tokens used in the NumberInput are following the new token naming
- [ ] Dev: All tokens used in the NumberInput are following the new token naming

----
#### Background information
- [Link](https://www.figma.com/file/C4vgEKz8mKyulJ4gm3Qdql/%F0%9F%AB%A7-%5BBLR%5D-The-B01LER?type=design&node-id=8035%3A7817&mode=design&t=eTH1Q2XmAvBy562Q-1) to new token naming documentation
- #357 should be done for this component in parallel

**Comment by larserbach** [comment]
Task: rename "Input" into "InputField"

**Comment by thrbnhrtmnn** [comment]
Hey @larserbach , do you mean we need a separate task to rename the component TextInput to TextInputField and NumberInput into NumberInputField?

**Comment by larserbach** [comment]
@thrbnhrtmnn my comment was meant as a reminder for me, what needs to be fixed/renamed in our tokens studio. Not releated to any other ticket.
